easybuild/easyconfigs/a/augur/augur-7.0.2-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,147 @@
+    }),
+    ('ipython-genutils', '0.2.0', {
+        'modulename': 'ipython_genutils',
+        'source_tmpl': 'ipython_genutils-%(version)s.tar.gz',
+        'checksums': ['eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8'],
+    }),
+    ('jsonschema', '3.2.0', {
+        'checksums': ['c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a'],
+    }),
+    ('jupyter-core', '4.6.3', {
+        'modulename': 'jupyter_core',
+        'source_tmpl': 'jupyter_core-%(version)s.tar.gz',
+        'checksums': ['394fd5dd787e7c8861741880bdf8a00ce39f95de5d18e579c74b882522219e7e'],
+    }),
+    ('kiwisolver', '1.2.0', {
+        'checksums': ['247800260cd38160c362d211dcaf4ed0f7816afb5efe56544748b21d6ad6d17f'],
+    }),
+    ('nbformat', '5.0.5', {
+        'checksums': ['f0c47cf93c505cb943e2f131ef32b8ae869292b5f9f279db2bafb35867923f69'],
+    }),
+    ('packaging', '20.3', {
+        'checksums': ['3c292b474fda1671ec57d46d739d072bfd495a4f51ad01a055121d81e952b7a3'],
+    }),
+    ('phylo-treetime', '0.7.5', {
+        'modulename': 'treetime',
+        'checksums': ['8e3b8003b75ca29b7e383d2ae0eb61dd47a6593eefdbb1e9995d93174b2ed576'],
+    }),
+    ('pyrsistent', '0.16.0', {
+        'checksums': ['28669905fe725965daa16184933676547c5bb40a5153055a8dee2a4bd7933ad3'],
+    }),
+    ('PyYAML', '5.3.1', {
+        'modulename': 'yaml',
+        'checksums': ['b8eac752c5e14d3eca0e6dd9199cd627518cb5ec06add0de9d32baeee6fe645d'],
+    }),
+    ('ratelimiter', '1.2.0.post0', {
+        'checksums': ['5c395dcabdbbde2e5178ef3f89b568a3066454a6ddc223b76473dac22f89b4f7'],
+    }),
+    ('seaborn', '0.9.1', {
+        'checksums': ['da33aa8c20a9a342ce73831d02831a10413f54a05471c7f31edf34f225d456ae'],
+    }),
+    ('smmap', '3.0.2', {
+        'checksums': ['b46d3fc69ba5f367df96d91f8271e8ad667a198d5a28e215a6c3d9acd133a911'],
+    }),
+    ('snakemake', '5.10.0', {
+        'checksums': ['a3fa8b12db84938c919996d61be66031bcb99c4b4d017278731324a6112b0d59'],
+    }),
+    ('toposort', '1.5', {
+        'checksums': ['dba5ae845296e3bf37b042c640870ffebcdeb8cd4df45adaa01d8c5476c557dd'],
+    }),
+    ('traitlets', '4.3.3', {
+        'checksums': ['d023ee369ddd2763310e4c3eae1ff649689440d4ae59d7485eb4cfbbe3e359f7'],
+    }),
+    ('wrapt', '1.12.1', {
+        'checksums': ['b62ffa81fb85f4332a4f609cab4ac40709470da05643a082ec1eb88e6d9b97d7'],
+    }),
+    (name, version, {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/nextstrain/augur/archive/'],
+        'use_pip_extras': 'full',
+        'checksums': ['bf520dfc7fe2bf72e12c8de25cd97ca84686345fbb1c4a3456b060500c9c25b3'],
+        # Change augur strict dependency on matplotlib and cvxopt. Newer versions can be used.
+        'preinstallopts': """sed -i'' 's/"matplotlib >=2.0, ==2.*",/"matplotlib >=2.0",/' setup.py &&""" +
+                          """sed -i'' 's/"cvxopt >=1.1.9, ==1.1.*",/"cvxopt >=1.1.9",/' setup.py &&
```

Comment:
@Darkless012 It's probably better to flesh this out into a local variable above.

I would shorted the `sed` commands a bit, to avoiding having to escape the literal `.*`:

```python
local_augur_preinstallopts = "sed -i'' 's/"matplotlib >=2.0,.*/"matplotlib >=2.0",/' setup.py && "
local_augur_preinstallopts += "sed -i'' 's/"cvxopt >=1.1.9,.*/"cvxopt >=1.1.9",/' setup.py && "

exts_list = [
    (name, version, {
        ...
        'preinstallopts': local_augur_preinstallopts,
```